brainzutils/musicbrainz_db/release_group.py

```diff
@@ -163,3 +163,55 @@ def _get_release_groups_for_artist_query(db, artist_id, release_types):
         join(models.ArtistCreditName, models.ArtistCreditName.artist_credit_id == models.ReleaseGroup.artist_credit_id).\
         join(models.Artist, models.Artist.id == models.ArtistCreditName.artist_id).\
         filter(models.Artist.gid == artist_id).filter(models.ReleaseGroupPrimaryType.name.in_(release_types))
+
+
+def get_release_groups_for_label(label_id, release_types=None, limit=None, offset=None):
+    """Get all release groups linked to a label.
+
+    Args:
+        label_id (uuid): MBID of the label.
+        release_types (list): List of types of release groups to be fetched.
+        limit (int): Max number of release groups to return.
+        offset (int): Offset that can be used in conjunction with the limit.
+
+    Returns:
+        Tuple containing the list of dictionaries of release groups ordered by release year
+        and the total count of the release groups.
```

Comment:
I'm just looking at the `case` that you have for order by, it looks like you put items without a release year last, is that right? If so, add that explicitly to this documentation

Reply:
Done